client/components/ReviewForm.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react'
+import { Redirect } from 'react-router-dom';
+import axios from 'axios'
+import { minReviewLength, maxReviewLength } from '../../server/constants'
+
+export default class ReviewForm extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      rating: 1,
```

Comment:
It's a little weird to see that the review starts out as one star. Can we swap this to 0?

Reply:
We could, although can you typically leave a 0 star review? I sort of expect 1 to 5, but I'm not dogmatic about it. Maybe we'll all be in a democratic mood tonight and can vote.

Reply:
I was thinking you don't allow them to submit with a 0-star review, but it is initialized that way

Reply:
that makes sense to me, I'll see about putting that in